nucliadb_ingest/nucliadb_ingest/processing.py

```diff
@@ -216,6 +216,28 @@ async def convert_filefield_to_str(self, file: models.FileField) -> str:
             jwttoken = await resp.text()
         return jwttoken
 
+    def convert_external_filefield_to_str(
+        self, file_field: models.ExternalFileField
+    ) -> str:
+        if self.nuclia_jwt_key is None:
+            raise AttributeError()
+
+        now = datetime.datetime.now(tz=datetime.timezone.utc)
+        expiration = now + datetime.timedelta(days=self.days_to_keep)
+        payload = {
+            "iss": "urn:nucliadb",
+            "sub": "file",
+            "aud": "urn:proxy",
+            "iat": now,
+            "exp": expiration,
+            "jti": uuid.uuid4().hex,
+            "source": 1,  # To indicate that this files comes internally
+            "driver": models.CloudFile.Source.EXTERNAL.value,
+            "uri": file_field.uri,
+            "extra_headers": file_field.extra_headers,
+        }
+        return jwt.encode(payload, self.nuclia_jwt_key, algorithm="HS256")
```

Comment:
@sunbit FYI: This is what is inside of the jwt that I'll be sending along on the `PushPayload` to the processing-slow